client/modules/IDE/hooks/useP5Version.jsx

```diff
@@ -144,7 +144,12 @@ export const p5Versions = [
 
 export const currentP5Version = '1.11.5'; // Don't update to 2.x until 2026
 
-export const p5SoundURLOld = `https://cdnjs.cloudflare.com/ajax/libs/p5.js/1.11.3/addons/p5.sound.min.js`;
+export const p5SoundURLOldTemplate =
+  'https://cdnjs.cloudflare.com/ajax/libs/p5.js/$VERSION/addons/p5.sound.min.js';
+export const p5SoundURLOld = p5SoundURLOldTemplate.replace(
+  '$VERSION',
+  '1.11.3'
```

Comment:
@davepagurek Would it be possible to use latest within 1.x? Since we are still bugfixing for a while.

Reply:
Updated this to use `currentP5Version`. I also updated the default web editor version to 1.11.8, since it was still 1.11.7 elsewhere. @raclim ideally we use the same string and splice it into `createDefaultFiles.js` so we have less places to update when there's a new version, as currently the two were out of date. Looks like that file is run on the server? Is there a spot for anything currently that gets imported in both client and server that I could move that constant to?

Reply:
I...don't believe that it exists yet. I'm sorry I think it might have to be created!

Reply:
ok I think I've added that! i added a thing to the dockerfile about the `common` folder, which seems to work running the project via docker, but I'm less confident that that's correct or necessary, so let me know if that seems ok to you!